packages/client/src/i18n/messages/form.ts

```diff
@@ -658,7 +659,12 @@ export const formMessageDescriptors: IFormMessages = {
   causeOfDeathEstablished: {
     defaultMessage: 'Has a cause of death been established ?',
     description: 'Label for form field: Cause of Death Established',
-    id: 'form.field.label.causeOfDeathEstablished'
+    id: 'form.field.label.CauseOfDeathEstablished'
+  },
+  causeOfDeathEstablishedMessages: {
+    defaultMessage: 'Cause of death has been established',
+    description: 'Has a cause of death been established',
+    id: 'form.field.label.CauseOfDeathEstablishedMessages'
```

Comment:
```suggestion
  causeOfDeathEstablishedFieldLabel: {
    defaultMessage: 'Cause of death has been established',
    description: 'Label for cause of death been established field',
    id: 'form.field.label.CauseOfDeathEstablishedFieldLabel'
```
Let's try to make the intention of the message a bit more explicit

Reply:
Alright